backend/homework/1/app/src/main/java/org/handson/Main.java

```diff
@@ -0,0 +1,90 @@
+package org.handson;
+
+
+
+public class Main {
+    Student getStudentDetail() {
+
+        Student student = new Student();
+
+
+        int id = 1;
```

Comment:
Instead of doing like this please create the student object and initialise the attributes in a constructor. Also `getStudentDetail()` should either be a simple getter or `getStudentDetailsById(id)`